### DISCUSSION TITLE
Inspector variables naming when name starts with "m_"

### DISCUSSION THREAD

**Discussion by Clayrok** [godotengine/godot-proposals#7780] [Editor]
Hi,

I've switched from Unity to Godot.
I'm used to call my member variables "m_MyVariable" (don't ask why, old habit haha).
In Unity the variable would automatically get renamed "My Variable" in the editor.
Here it is renamed "M My Variable" which looks weird.

**Comment by AThousandShips** [comment]
This is really a limitation you have to get used to I'm afraid, we can't really add exceptions for every possible prefix or style, the style for member variables in Godot projects is to prefix with `_`

**Reply by Clayrok** [reply]
Sure, not a big deal. I can live with that :)

**Reply by AThousandShips** [reply]
See [here](https://docs.godotengine.org/en/stable/tutorials/scripting/c_sharp/c_sharp_style_guide.html#naming-conventions) for style conventions in C#, these inform how the editor and engine handle things :)

**Reply by Clayrok** [reply]
Thanks a lot!

**Comment by Calinou** [comment]
I think supporting `m_` (member), `u_` (shader uniform) and `p_` (parameter) prefixes is something we could do, but there will probably be some false positives. We don't want every single-letter prefix to be dropped from the editor representation, as it's sometimes relevant (e.g. `x_scale` for X axis scale).

Personally, I use `p_` in GDScript for function parameters to match what the engine does in C++. It makes it a lot easier to distinguish function parameters from local or member variables at a glance, and can prevent you from accidentally assigning them.

`u_` prefix is also sometimes seen in GLSL shaders.

**Reply by Clayrok** [reply]
Maybe an option could be added in the editor settings so each individual can choose what prefixes they want to filter?

**Reply by Calinou** [reply]
I don't think the text displayed for inspector properties should vary depending on editor settings. This can lead to confusing situations when someone posts a screenshot in a tutorial or forum post, and their property names look nothing like other people's.
